openlcs/libs/tests.py

```diff
@@ -371,56 +371,57 @@ def setUp(self):
+        self.corgi_connector = CorgiConnector(base_url)
 
     @mock.patch.object(
-        ParentComponentsAsync, 'get_component_data_from_corgi')
+        CorgiConnector, 'get_component_data_from_corgi')
     def test_get_component_data_1(
             self, mock_get_component_data_from_corgi):
         mock_get_component_data_from_corgi.return_value = \
             self.components_data[0]
-        component = self.container_components.get_component_data(
+        component = self.corgi_connector.get_component_data(
             self.links[0])
         self.assertEqual(component, self.components_data[0])
 
     @mock.patch.object(
-        ParentComponentsAsync, 'get_component_data_from_corgi')
-    @mock.patch.object(ParentComponentsAsync, 'parse_component_link')
+        CorgiConnector, 'get_component_data_from_corgi')
+    @mock.patch.object(CorgiConnector, 'parse_component_link')
     def test_get_component_data_2(
             self, mock_get_component_data_from_corgi,
             mock_parse_component_link):
         mock_get_component_data_from_corgi.return_value = {}
         mock_parse_component_link.return_value = \
             self.components_data[1]
-        component = self.container_components.get_component_data(
+        component = self.corgi_connector.get_component_data(
             self.links[1])
         self.assertEqual(component, self.components_data[1])
 
     @mock.patch.object(
-        ParentComponentsAsync, 'get_component_data_from_corgi')
-    @mock.patch.object(ParentComponentsAsync, 'parse_component_link')
+        CorgiConnector, 'get_component_data_from_corgi')
+    @mock.patch.object(CorgiConnector, 'parse_component_link')
     def test_get_component_data_3(
             self, mock_get_component_data_from_corgi,
             mock_parse_component_link):
         mock_get_component_data_from_corgi.return_value = {}
         mock_parse_component_link.return_value = \
             self.components_data[2]
-        component = self.container_components.get_component_data(
+        component = self.corgi_connector.get_component_data(
             self.links[2])
         self.assertEqual(component, self.components_data[2])
 
-    @mock.patch.object(ParentComponentsAsync, 'get_event_loop')
-    @mock.patch.object(ParentComponentsAsync, 'get_component_and_links')
+    @mock.patch.object(CorgiConnector, 'get_event_loop')
+    @mock.patch.object(CorgiConnector, 'get_component_and_links')
     def test_get_components_data(self, mock_get_component_and_links,
                                  mock_get_event_loop):
         mock_get_component_and_links.return_value = \
             self.links, self.components_data[3]
         mock_get_event_loop.return_value = self.components_data[:-1]
         self.assertEqual(
-            self.container_components.get_components_data('CONTAINER_IMAGE'),
+            self.corgi_connector.get_components_data(
+                self.nvr,
+                'CONTAINER_IMAGE'),
```

Comment:
The type 'CONTAINER_IMAGE' could be updated to 'OCI' since the type has been updated to 'OCI' on corgi production.

Reply:
rebase fixed it.